pom.xml

```diff
@@ -258,7 +254,7 @@
                     </configuration>
                     <executions>
                         <execution>
-                            <id>mojo-descriptor</id>
+                            <id>default-descriptor</id>
                             <phase>process-classes</phase>
```

Comment:
Not nocessary, but will be as soon as we update this mojo: in later versions a default execution with this name is present.